### ISSUE TITLE
val_loss stuck at zero while running TabTransformer fine tuning code

### ISSUE THREAD

**Issue by sathwikacharya** [aruberts/TabTransformerTF#1]
Hello,
I am following the tabtransformer pre-training notebook found in the github repo. While fine tuning, I pass in my validation set as you can see in the image below. I noticed that val_loss and val_AUC scores never gets updated. Can you suggest a possible fix for this? This issue might be related to the one mentioned here: https://github.com/tensorflow/tensorflow/issues/39370

Thank you

<img width="1082" alt="Screenshot 2022-11-16 at 12 06 30 PM" src="https://user-images.githubusercontent.com/60059843/202103793-4213af9d-5603-4c07-bf5e-1977742afe87.png">
